lib/extensions/logging_extensions.rb

```diff
@@ -0,0 +1,41 @@
+module LoggingExtensions
+  # Create our default log formatter so that we can use it everywhere, and keep formats consistent.
+  def self.default_log_formatter
+    @default_log_formatter = if Rails.env.development? || Rails.env.profiling? || Rails.env.test?
+      Ougai::Formatters::Readable.new
+    else
+      Ougai::Formatters::Bunyan.new
```

Comment:
This ensures we always get a JSON style formatter w/ the correct extensions added.